app/timedatectl/doc/README.head.md

```diff
@@ -0,0 +1 @@
+High level monitoring for time and date keeping based on the backend agnostic timedatectl command.
```

Comment:
How about linking to the Systemd timedatectl command documentation?

> [...] based on the backend agnostic [timedatectl](https://www.freedesktop.org/software/systemd/man/timedatectl.html) command.

Reply:
link added